rfc/rfc020-authorization-credential-extension.md

```diff
@@ -0,0 +1,64 @@
+# RFC020 Authorization credential extension
+
+|                           |                 |
+|:--------------------------|:----------------|
+| Nuts foundation           | W.M. Slakhorst  |
+| Request for Comments: 020 | Nedap           |
+| Ammends: RFC014           | April 2023      |
+
+## Authorization credential extension
+
+### Abstract
+
+A `assuranceLevel` field is added to the `NutsAuthorizationCredential`. It can be used within the context of a `resource` to indicate the required assurance level of the authentication.
```

Comment:
You're referring to technical fields, so be precise ("context" makes things vague)

```suggestion
A `assuranceLevel` field is added to the `NutsAuthorizationCredential`. It can be used inside a `resource` to indicate the required assurance level of the authentication.
```